osfoffline/filesystem_manager/osf_event_handler.py

```diff
@@ -229,7 +229,6 @@ def on_deleted(self, event):
     def dispatch(self, event):
         # basically, ignore all events that occur for 'Components' file or folder
         if self._event_is_for_components_file_folder(event):
```

Comment:
A more elegant way would be to check the event type. The error occurs because when osf offline creates the folder structure, it detects its own create/modified event.